Added a callout about -m flag (#273)
carpentries-incubator/python-intermediate-development: _episodes/12-virtual-environments.md

- "`pip` allows you to declare all dependencies for a project in a separate
file (by convention called `requirements.txt`) which can be shared with collaborators/users and used to replicate a virtual environment."
- "Use `pip3 freeze > requirements.txt` to take snapshot of your project's dependencies."

> - "Use `pip3 install -r requirements.txt` to replicatefpython3 -m someone else's virtual environment on your machine from

Reviewer: Probably needs correction :)

Author: fixed via to our virtual environment

> - "Use `pip3 install -r requirements.txt` to replicatefpython3 -m someone else's virtual environment on your machine from

Author: ```suggestion
- "Use `pip3 install -r requirements.txt` to replicate someone else's virtual environment on your machine from
```

This will create the target directory for the virtual environment
(and any parent directories that don’t exist already).

> ## What is `-m` Flag in `python3` Command??
> The Python `-m` flag means "module" and tells the Python interpreter to treat what follows `-m` 
> as the name of a module and not as a single, executable program with the same name. Some modules 
> (such as `venv` or `pip`) have main entry points and the `-m` flag can be used to invoke them 
> on the command line via the `python` command. The main difference between running such modules as 
> standalone programs (e.g. executing "venv" by running the `venv` command directly) 
> versus using `python3 -m` command seems to be that with latter you are in full control of 

> > which `venv` module will be invoked (the one that came with your Python interpreter vs. 

Author: ```suggestion
> which Python module will be invoked (the one that came with your Python interpreter vs. 
```

> > which `venv` module will be invoked (the one that came with your Python interpreter vs. 

Reviewer: ```suggestion
> which Python module will be invoked (the one that came with your Python interpreter vs. 
```

Author: fixed via 7ee460d2025cdacf3223f985758c08408588e3f9

> > which Python module will be invoked (the one that came with your Python interpreter vs. 

Author: ```suggestion
> which Python module will be invoked (the one that came with your environment's Python interpreter vs. 
```
